lib/djvAV/PFM.cpp

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2004-2020 Darby Johnston
+// All rights reserved.
+
+#include <djvAV/PFM.h>
+
+using namespace djv::Core;
+
+namespace djv
+{
+    namespace AV
+    {
+        namespace IO
+        {
+            namespace PFM
+            {
+                struct Plugin::Private
+                {};
+
+                Plugin::Plugin() :
+                    _p(new Private)
+                {}
+
+                std::shared_ptr<Plugin> Plugin::create(const std::shared_ptr<System::Context>& context)
+                {
+                    auto out = std::shared_ptr<Plugin>(new Plugin);
+                    out->_init(
+                        pluginName,
+                        DJV_TEXT("plugin_pfm_io"),
```

Comment:
A text string should be added for this in the file "etc/Text/djvAV.en.text"; for example the text string for the RLA plugin is, "This plugin provides RLA image I/O."

After this is checked in I will run the translation scripts that will populate the other language files with this text string.